src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -756,7 +754,7 @@ FixedwingPositionControl::set_control_mode_current(const hrt_abstime &now, bool
 	_skipping_takeoff_detection = false;
 
 	if (((_control_mode.flag_control_auto_enabled && _control_mode.flag_control_position_enabled) ||
-	     _control_mode.flag_control_offboard_enabled) && pos_sp_curr_valid) {
```

Comment:
Could we take this from the parameter? I was trying to make the behavior of the logic more deterministic by not relying on a global variable of the position setpoint triplet